### PR TITLE
William/Updated how to buy page

### PR DESCRIPTION
**Pull request by swordman1205**
What I have done:
- PSD to HTML layout
- Responsive web design
- Combine my work into code base

### CODE REVIEW

**Comment by swordman1205** [comment]
Hi @RassanGrant, please check my work. Thanks!

**Comment by albertvolkman** [comment]
Please follow the same formatting guidelines as the rest of the site (no tabs, two spaces). Do not create a new import... as you can see, the site is using SMACSS. We will not be creating a new sass file for individual pages.

File: _sass/_howtobuy.scss [hunk 1]
```diff
@@ -0,0 +1,442 @@
+*{
```
Comment (albertvolkman) [inline _sass/_howtobuy.scss]: Please stay away from global resets, as Foundation is already doing this.

**Comment by albertvolkman** [comment]
You're introducing a lot of layout-specific CSS. Please utilize more of Foundation's so that we don't have to fight making the site mobile-optimized.

File: how-to-buy.html [hunk 2]
```diff
@@ -7,8 +7,64 @@
...
+		</div>
+
+		<div class="rightSubContainer">
+			<div class="contactEmail">Shoot us an email</div>
+			<div class="required"><p>* <span>Required Fields</span></p></div>
+			{% include contact-form.html %}
+		</div>	
+	</div>
+			
+</div>
```
Comment (swordman1205) [inline how-to-buy.html]: I am not sure what you mean.

Comment (albertvolkman) [inline how-to-buy.html]: https://robots.thoughtbot.com/no-newline-at-end-of-file

**Comment by albertvolkman** [comment]
There's still some formatting issues with your SASS. Please run SASSLint against your work-

https://www.npmjs.com/package/sass-lint

File: _sass/_layout.scss [hunk 3]
```diff
@@ -277,3 +277,342 @@
 #select-color {
   margin-top: 17px;
 }
+
+// how-to-buy layout
+
+@font-face{
```
Comment (albertvolkman) [inline _sass/_layout.scss]: Typography belongs in https://github.com/Norstone/norstone.github.io/blob/develop/_sass/_typography.scss

File: _sass/_layout.scss [hunk 4]
```diff
@@ -277,3 +277,342 @@
...
+.container {
+  background-color: #eaeaea;
+  margin: auto;
+  width: 82%;
+  max-width: 990px;
+  text-align: center;
+  display: table;
+  margin-top: 10px;
+  margin-bottom: 10px;
+
```
Comment (albertvolkman) [inline _sass/_layout.scss]: Extra space.

File: _sass/_layout.scss [hunk 5]
```diff
@@ -277,3 +277,342 @@
...
+  text-align: center;
+  display: inline-block;
+}
+
+.topContainer {
+  position: relative;
+  z-index: 2;
+  margin-top: -20px;
+}
+.infoContainer {
```
Comment (albertvolkman) [inline _sass/_layout.scss]: Please add a blank line between declarations.

File: _sass/_layout.scss [hunk 6]
```diff
@@ -277,3 +277,342 @@
...
+  font-weight: bold;
+  padding: 0;
+}
+
+.required p > span{
+  color : #5a5959;
+  font-size: 13px;
+}
+
+@media screen and (max-width: 1094px){
```
Comment (albertvolkman) [inline _sass/_layout.scss]: Use Foundation's SASS breakpoints-

http://foundation.zurb.com/sites/docs/media-queries.html

Example here-

https://github.com/Norstone/norstone.github.io/blob/develop/_sass/_menu.scss#L1

File: _sass/_layout.scss [hunk 7]
```diff
@@ -277,3 +277,342 @@
...
+@media screen and (max-width: 480px){
+  .contactEmail {
+    font-size: 28px;
+  }
+  .middle p{
+    font-size: 20px;
+  }
+
+}
+
```
Comment (albertvolkman) [inline _sass/_layout.scss]: Extraneous blank line.

File: how-to-buy.html [hunk 8]
```diff
@@ -7,8 +7,70 @@
...
+    </div>
+
+    <div class="large-6 columns">
+      <div class="contactEmail">Shoot us an email</div>
+      <div class="required"><p>* <span>Required Fields</span></p></div>
+      {% include contact-form.html %}
+    </div>  
   </div>
-</div>
+
```
Comment (albertvolkman) [inline how-to-buy.html]: Extraneous blank line, still missing EOL.

File: how-to-buy.html [hunk 9]
```diff
@@ -7,8 +7,70 @@
...
 ---
 
-<div class="row" style="background-size: 100%; background-image: url(/assets/images/unsorted/How-to-Buy-InfoGraphic_1075pxV2-Orange.png); background-repeat: no-repeat; margin: 10px auto; padding-top: 430px">
-  <div class="large-7 large-offset-5 columns" style="padding: 0 40px 20px">
-    {% include contact-form.html %}
+ <div class="container">
+  <p class="titleContainer">Norstone is available Nationwide.</p>
+  <div class="topContainer">
+    <div class="row infoContainer">
+      <div class="large-4 columns ">
```
Comment (albertvolkman) [inline how-to-buy.html]: Extra space after class.

File: how-to-buy.html [hunk 10]
```diff
@@ -7,8 +7,70 @@
...
+        <div class="bgView">
+          <h3>Homeowners</h3>
+          <p>Large or small, <span class="blue">Every Project is Important to us.</span> Let our staff or our carefully selected local distributors help make your dream a reality.</p>
+        </div>
+      </div>
+
+      <div class="large-4 columns ">
```
Comment (albertvolkman) [inline how-to-buy.html]: Extra space after class.

File: how-to-buy.html [hunk 11]
```diff
@@ -7,8 +7,70 @@
...
+  <p class="titleContainer">Norstone is available Nationwide.</p>
+  <div class="topContainer">
+    <div class="row infoContainer">
+      <div class="large-4 columns ">
+        <div class="bgView">
+          <h3>Homeowners</h3>
+          <p>Large or small, <span class="blue">Every Project is Important to us.</span> Let our staff or our carefully selected local distributors help make your dream a reality.</p>
+        </div>
+      </div>
+
```
Comment (albertvolkman) [inline how-to-buy.html]: Extraneous blank line.

File: how-to-buy.html [hunk 12]
```diff
@@ -7,8 +7,70 @@
...
+      </div>
+      <div class="contact">
+        <div class="call">GIVE US A CALL</div>
+        <div class="number">
+          <div class="overlay"></div>
+          <span>(866)<br>717<br>4548</span>
+        </div>
+      </div>
+    </div>
+
```
Comment (albertvolkman) [inline how-to-buy.html]: Extraneous blank line.

File: _sass/_layout.scss [hunk 13]
```diff
@@ -277,3 +277,342 @@
...
+}
+.requestItems p > span:first-child{
+  font-size: 50px;
+  color: #3aa4cb;
+  text-shadow: 1px 1px 1px #000;
+  padding: 0 4px;
+  position: relative;
+  top: 6px;
+}
+.itemsContainder {
```
Comment (albertvolkman) [inline _sass/_layout.scss]: Spelling error.

**Comment by albertvolkman** [comment]
Are you familiar with SASS's nesting capabilities? It doesn't appear so, per your current implementation.

**Comment by swordman1205** [comment]
Hi @RassanGrant, @albertvolkman 
Please check a new update.

**Comment by albertvolkman** [comment]
@swordman1205 you are defining several base-level elements, namely-
````
body, h1, h3, p
````
Have you verified that these have no effect on the rest of the site? I have not checked personally, but @RassanGrant has noticed several issues...

**Comment by RassanGrant** [comment]
@swordman1205 - The issue I'm seeing is site-wide in the header (see attached). If we define body, h1, h3, and p as page-specific elements, will that fix the issue?

2016-10-13-05 09-edt

**Comment by swordman1205** [comment]
Hi, @RassanGrant I fixed the header layout issue. Please check it.

**Comment by RassanGrant** [comment]
Hi, William - Thanks for your latest round of code. I am not seeing the anomaly in the header any longer. I would like to adjust the content types to be as as follows:

how-to-buy-feedback

**Comment by swordman1205** [comment]
Hi, @RassanGrant I have changed the content types you pointed. Please check the code.

**Pull request merged**